delivery_carrier_label_batch/data/ir.config_parameter.xml

```diff
@@ -1,10 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <odoo noupdate="1">
-    <record id="zpl2_assembler_single_images" model="ir.config_parameter">
+    <record
+        forcecreate="True"
```

Comment:
Why adding `forcecreate`? 
I'm asking because it is unclear to me when to use it

Reply:
As I understand if the record doesn't exist `forcecreate` ensures it will be created. I think to use the module features we need parameter to be 100% created.